R/pkg/R/generics.R

```diff
@@ -424,6 +424,14 @@ setGeneric("cov", function(x, col1, col2) {standardGeneric("cov") })
 #' @export
 setGeneric("corr", function(x, ...) {standardGeneric("corr") })
 
+#' @rdname statfunctions
+#' @export
+setGeneric("covar_samp", function(x, ...) {standardGeneric("covar_samp") })
```

Comment:
The number of parameters are fixed to be 2. No need to support varargs.
Something like:
setGeneric("covar_samp", function(col1, col2) {standardGeneric("covar_samp") })